test/utilities/oas-operation/oas-operation.test.ts

```diff
@@ -164,4 +167,20 @@ describe('OASOperation', () => {
       expect(operation.getResponseSchema(683)).toBeNull();
     });
   });
+
+  describe('getSecurity', () => {
+    it('uses operation level security if it exists', () => {
```

Comment:
Not sure if this suggestion is accurate

```suggestion
    it('returns operation level security if it exists', () => {
```

Reply:
fixed this whole section. thanks!